tests/test_models.py

```diff
@@ -1309,13 +1314,75 @@ def test_string_conversion(self, method):
         """
         Test ``SAPSuccessFactorsGlobalConfiguration`` conversion to string
         """
-        config = SAPSuccessFactorsGlobalConfiguration(
-            id=2,
-            completion_status_api_path='completion_status',
-            course_api_path='courses',
-            oauth_api_path='oauth'
+        config = factories.SAPSuccessFactorsGlobalConfigurationFactory(id=1)
+        expected_to_str = "<SAPSuccessFactorsGlobalConfiguration with id 1>"
+        assert expected_to_str == method(config)
+
+
+@mark.django_db
+@ddt.ddt
+class TestDegreedLearnerDataTransmissionAudit(unittest.TestCase):
+    """
+    Tests of the ``DegreedLearnerDataTransmissionAudit`` model.
+    """
+
+    @ddt.data(str, repr)
+    def test_string_conversion(self, method):
+        """
+        Test ``DegreedLearnerDataTransmissionAudit`` conversion to string
+        """
+        learner_audit = factories.DegreedLearnerDataTransmissionAuditFactory(
+            id=1,
+            enterprise_course_enrollment_id=5,
+            degreed_user_id='degreed_user',
+            course_id='course-v1:edX+DemoX+DemoCourse',
         )
-        expected_to_str = "<SAPSuccessFactorsGlobalConfiguration with id 2>"
+        expected_to_str = (
+            "<DegreedLearnerDataTransmissionAudit 1 for enterprise enrollment 5, Degreed user degreed_user,"
+            " and course course-v1:edX+DemoX+DemoCourse>"
```

Comment:
@UmanShahzad Nit: Include space at the end of the previous line.

Reply:
:+1: